integration-tests/testconfig/common/vrf/common.go

```diff
@@ -0,0 +1,297 @@
+		return errors.New("subscription_funding_amount_link must be set to a positive value")
+	}
+	if c.NumberOfWords == nil || *c.NumberOfWords == 0 {
+		return errors.New("number_of_words must be set to a positive value")
+	}
+	if c.CallbackGasLimit == nil || *c.CallbackGasLimit == 0 {
+		return errors.New("callback_gas_limit must be set to a positive value")
+	}
+	if c.MaxGasLimitCoordinatorConfig == nil || *c.MaxGasLimitCoordinatorConfig == 0 {
+		return errors.New("max_gas_limit_coordinator_config must be set to a positive value")
+	}
+	if c.FallbackWeiPerUnitLink == nil || *c.FallbackWeiPerUnitLink == 0 {
+		return errors.New("fallback_wei_per_unit_link must be set to a positive value")
+	}
+	if c.StalenessSeconds == nil || *c.StalenessSeconds == 0 {
+		return errors.New("staleness_seconds must be set to a positive value")
+	}
+	if c.GasAfterPaymentCalculation == nil || *c.GasAfterPaymentCalculation == 0 {
+		return errors.New("gas_after_payment_calculation must be set to a positive value")
+	}
+	if c.NumberOfSubToCreate == nil || *c.NumberOfSubToCreate == 0 {
+		return errors.New("number_of_sub_to_create must be set to a positive value")
+	}
+	if c.RandomnessRequestCountPerRequest == nil || *c.RandomnessRequestCountPerRequest == 0 {
+		return errors.New("randomness_request_count_per_request must be set to a positive value")
+	}
+	if c.RandomnessRequestCountPerRequestDeviation == nil {
+		return errors.New("randomness_request_count_per_request_deviation must be set to a non-negative value")
+	}
+	if c.RandomWordsFulfilledEventTimeout == nil || c.RandomWordsFulfilledEventTimeout.Duration == 0 {
+		return errors.New("random_words_fulfilled_event_timeout must be set to a positive value")
+	}
+	if c.WrapperGasOverhead == nil {
+		return errors.New("wrapped_gas_overhead must be set to a non-negative value")
+	}
+	if c.CoordinatorGasOverhead == nil || *c.CoordinatorGasOverhead == 0 {
+		return errors.New("coordinator_gas_overhead must be set to a non-negative value")
+	}
+	if c.WrapperPremiumPercentage == nil || *c.WrapperPremiumPercentage == 0 {
+		return errors.New("wrapper_premium_percentage must be set to a positive value")
+	}
+	if c.WrapperMaxNumberOfWords == nil || *c.WrapperMaxNumberOfWords == 0 {
+		return errors.New("wrapper_max_number_of_words must be set to a positive value")
+	}
+	if c.WrapperConsumerFundingAmountNativeToken == nil || *c.WrapperConsumerFundingAmountNativeToken < 0 {
+		return errors.New("wrapper_consumer_funding_amount_native_token must be set to a non-negative value")
+	}
+	if c.WrapperConsumerFundingAmountLink == nil || *c.WrapperConsumerFundingAmountLink < 0 {
+		return errors.New("wrapper_consumer_funding_amount_link must be set to a non-negative value")
+	}
+	if *c.RandomnessRequestCountPerRequest <= *c.RandomnessRequestCountPerRequestDeviation {
+		return errors.New(ErrDeviationShouldBeLessThanOriginal)
+	}
+
+	if c.VRFJobForwardingAllowed == nil {
```

Comment:
nit peak: put job validation is separate methods so it's easier to read this one big method?

Reply:
in a separate pr i want to reorganize configs - have separate structs for Jobs, Node config, Contract configs